source/deployment/mobile-app-deployment.rst

```diff
@@ -251,19 +231,19 @@ This incompatibility is not an issue with the Mattermost application. It results
 In the end, this sets app wrapping in a negative light. This is the reason the `AppConfig Community <https://www.appconfig.org>`_ came together to create a standard. AppConfig is a modern, efficient, and scalable approach to enterprise mobile management. In that regard, the Mattermost application benefits from being built around this standard.
 
 ******************
-Mobile VPN Options
+Mobile VPN options
 ******************
 
 A Virtual Private Networks (VPN) allows a device outside a firewall to access content inside the firewall as if it were on the same network. Most enterprise teams are familiar with VPNs. We won't go into detail here.
 
-There are VPN options which depend on the requirements of your organization. You should also consider the demands/needs of your user. Regardless, this can impact your approach to mobile deployment.
+There are VPN options which depend on the requirements of your organization. You should also consider the demands/needs of your users. Regardless, this can impact your approach to mobile deployment.
 
-For the Mattermost mobile application, we’ll discuss two options: a device VPN or per-app VPN.
+For the Mattermost mobile application, we will discuss two options: a device VPN or per-app VPN.
 
     **Note:** We suggest `following our recommended steps <https://docs.mattermost.com/mobile/mobile-appstore-install.html>`_ to secure your deployment.
 
 **Device VPN**
-    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issue for personal applications. Of course even for devices provided by your business, this tends not to be the modern approach.
+    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issue for personal applications.
```

Comment:
```suggestion
    This is not as common, especially in the case of Bring Your Own Device (BYOD) scenarios. In this option, all internet traffic routes through the VPN specified in the profile. This could cause issues for personal applications.
```